tests/integration/integration_test.go

```diff
@@ -345,44 +346,44 @@ func genOperatorConfig(t *testing.T, ctx context.Context, mainnetAnvil *AnvilIns
 	os.Setenv("OPERATOR_BLS_KEY_PASSWORD", "")
 	os.Setenv("OPERATOR_ECDSA_KEY_PASSWORD", "")
 
-	err = os.MkdirAll(getOperatorKeysPathPrefix(t), 0770)
+	nodeConfig.BlsPrivateKeyStorePath, err = filepath.Abs(filepath.Join("../keys/bls", keysPath, "key.json"))
 	if err != nil {
-		t.Fatalf("Failed to create operators keys dir: %s", err.Error())
+		t.Fatalf("Failed to get BLS key dir: %s", err.Error())
 	}
-
-	keysPath, err := os.MkdirTemp(getOperatorKeysPathPrefix(t), "")
+	passwordPath := filepath.Join("../keys/bls", keysPath, "password.txt")
+	password, err := os.ReadFile(passwordPath)
 	if err != nil {
-		t.Fatalf("Failed to create operator keys dir: %s", err.Error())
+		t.Fatalf("Failed to read BLS password: %s", err.Error())
 	}
-
-	nodeConfig.BlsPrivateKeyStorePath, err = filepath.Abs(filepath.Join(keysPath, "test.bls.key.json"))
-	if err != nil {
-		t.Fatalf("Failed to get BLS key dir: %s", err.Error())
+	if string(password) != "" {
+		t.Fatalf("Password is not empty: '%s'", password)
 	}
-	keyPair, err := bls.GenRandomBlsKeys()
+	keyPair, err := bls.ReadPrivateKeyFromFile(nodeConfig.BlsPrivateKeyStorePath, "")
 	if err != nil {
 		t.Fatalf("Failed to generate operator BLS keys: %s", err.Error())
 	}
-	err = keyPair.SaveToFile(nodeConfig.BlsPrivateKeyStorePath, "")
-	if err != nil {
-		t.Fatalf("Failed to save operator BLS keys: %s", err.Error())
-	}
 
-	nodeConfig.EcdsaPrivateKeyStorePath, err = filepath.Abs(filepath.Join(keysPath, "test.ecdsa.key.json"))
+	nodeConfig.EcdsaPrivateKeyStorePath, err = filepath.Abs(filepath.Join("../keys/ecdsa", keysPath, "key.json"))
 	if err != nil {
 		t.Fatalf("Failed to get ECDSA key dir: %s", err.Error())
 	}
-	ecdsaKey, err := crypto.GenerateKey()
+	passwordPath = filepath.Join("../keys/ecdsa", keysPath, "password.txt")
```

Comment:
```suggestion
	passwordPath = filepath.Join(EcdsaKeysDir, keysPath, "password.txt")
```

Reply:
Done in [450c845](https://github.com/NethermindEth/near-sffl/pull/71/commits/450c845b4021af70c0480f82ae87e46af23bb686)